wolftpm/doxygen.h

```diff
@@ -0,0 +1,63 @@
+/** @defgroup wolfTPM2_Wrappers wolfTPM2 Wrappers
```

Comment:
Can these be included in the appropriate headers?

Reply:
Good tip. I am moving them